internal/pkg/aws/ecs/ecs.go

```diff
@@ -106,6 +107,26 @@ func New(s *session.Session) *ECS {
 	}
 }
 
+// GetTags list the tags for an Amazon ECS resource given its ARN.
+func (e *ECS) GetTags(arn string) (map[string]string, error) {
+	resp, err := e.client.ListTagsForResource(&ecs.ListTagsForResourceInput{
+		ResourceArn: aws.String(arn),
+	})
+	if err != nil {
+		return nil, err
+	}
+	tags := make(map[string]string)
+	for _, tag := range resp.Tags {
+		if tag == nil {
+			continue
+		}
+		if tag.Key != nil && tag.Value != nil {
+			tags[*tag.Key] = *tag.Value
+		}
```

Comment:
I think the key has to be not nil but the value can be nil :/
> Each tag consists of a key and an optional value, both of which you define. 
(https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html)

So 
```suggestion
		if tag.Key == nil {
			continue
		}
		key := *tag.Key
		value := ""
		if tag.Value != nil {
			value = *tag.Value
		}
```